### PR TITLE
Implement adding card with bloc

### PR DESCRIPTION
**Pull request by Pasan-Pahasara**
[ITP-1747]

[ITP-1747]: https://fcodelabs.atlassian.net/browse/ITP-1747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/app/google-services.json
Comment (KavinduDhananjaya) [inline android/app/google-services.json:1]: add this file to gitignore.

**Comment by Pasan-Pahasara** [comment]
done

File: lib/ui/login_page/login_page_view.dart [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+            ),
+          ),
+          body: Container(
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage('assets/img/background.jpg'),
+                fit: BoxFit.cover,
+              ),
+            ),
+            child: Scaffold(
```
Comment (KavinduDhananjaya) [inline lib/ui/login_page/login_page_view.dart]: dont use two scaffold in same widget.

File: lib/ui/login_page/login_page_view.dart [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+import 'package:training_app/ui/login_page/login_page_state.dart';
+
+class LoginScreenView extends StatelessWidget {
+  const LoginScreenView({Key? key}) : super(key: key);
+
+  void navigateToDiaryHomeScreen(BuildContext context, String name) {
+    Navigator.push(
+      context,
+      MaterialPageRoute(
+        builder: (context) => Scaffold(
```
Comment (KavinduDhananjaya) [inline lib/ui/login_page/login_page_view.dart]: create seperate widget and call in here.

**Pull request merged**